content/terraform/v1.12.x/docs/language/values/variables.mdx

```diff
@@ -1,29 +1,21 @@
 ---
-page_title: Set configuration parameters
-description: Learn how to make Terraform modules flexible, composable, and reusable by defining input variables, locally-scoped values, and output values for your module.
+page_title: Use input variables to add module arguments
+description: Learn how to define input variables so that you can input new values at runtime.
 ---
 
-# Set configuration parameters
+# Use input variables to add module arguments
 
-Make your Terraform modules flexible, composable, and reusable by defining your module's input variables, defining module-scoped local values, and defining output values to expose data from your module.
-
-## Overview
-
-Use the following blocks to modify your module's behavior:
-
-- The `variable` block lets you define input arguments, letting module consumers pass in values to customize behavior at runtime.
-- The `locals` block defines temporary values scoped to a module, letting you name and reuse expressions in your configuration.
-- The `outputs` block exposes data from a module, making module results available in the Terraform CLI, HCP Terraform, and other parts of your configuration.
+> **Hands-on:** Try the [Customize Terraform Configuration with Variables](/terraform/tutorials/configuration-language/variables?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) tutorial.
 
-## Define module input arguments with variables
+Variables define the input interface of your module by specifying what values your module accepts as arguments.
```

Comment:
```suggestion
You can add `variable` blocks to your configuration to define input interface for your module. This lets users pass custom values to your module at runtime.
```
Right? Ignore me if I'm wrong, but the original reads to me as though variables are provider developer-side construct. Well, they are in that the dev can program what variables the resources take, but I don't think that's what we intend to mean in this topic.